dot_lpdev_cmds.sh

```diff
@@ -642,9 +604,9 @@ function __lpdev_node_broadcaster {
   nodeDataDir=$nodeBaseDataDir/broadcaster-${broadcasterGeth:0:10}
   if [ ! -d $nodeDataDir ]
   then
-    mkdir -p $nodeDataDir/keystore
+    mkdir -p $nodeDataDir/devenv/keystore
```

Comment:
why do we need to add `devenv` to this path?

Reply:
I see this is related to the `-devenv` flag, so I'll update my question to: why is it useful to use the `-devenv` flag?

Reply:
> why is it useful to use the -devenv flag?

I added it because at the moment if you run the node without the `-devenv` flag, the node will default to connecting to mainnet.

Reply:
That makes sense, thanks for clarifying!